clients/es-integration/build.gradle

```diff
@@ -12,13 +13,45 @@ checkstyle {
     toolVersion "6.7"
 }
 
+idea {
```

Comment:
why did you add this settings section?

Reply:
Added to separate idea and gradle output directories